shap/plots/_beeswarm.py

```diff
@@ -823,7 +825,7 @@ def summary_legacy(shap_values, features=None, feature_names=None, max_display=N
                 y = ys[i, :] / scale
                 c = pl.get_cmap(color)(i / (
                         nbins - 1)) if color in pl.cm.datad else color  # if color is a cmap, use it, otherwise use a color
-                pl.fill_between(x_points, pos - y, pos + y, facecolor=c)
+                pl.fill_between(x_points, pos - y, pos + y, facecolor=c, edgecolor="face")
```

Comment:
This change is not really about F821. It's the fact that `pytest_mpl` plugin was generating plots with black edges (under the "classic" style) which covered the whole plot in black lines...

![image](https://github.com/dsgibbons/shap/assets/30731072/43d898a3-5946-4f69-bc89-6ce19030859c)

So it's better to be explicit here about the fact that we want no edges (here, I went with same color as facecolor).